src/user_utils.jl

```diff
@@ -23,6 +23,61 @@ function tonemapping(input_file::String,
+                         Sphere(transformation = translation([0f0, 0.5f0, 0f0]) * scaling(1f0/10),
+                                material = Material(brdf = DiffuseBRDF(pigment = ImagePigment(HdrImage([RED GREEN BLUE MAGENTA])))))]
+        ground = [Plane(transformation = translation([0f0, 0f0, -1f0]),
+                        material = Material(brdf = DiffuseBRDF(pigment = CheckeredPigment{4}())))]
+        sky = [Plane(transformation = translation([0f0, 0f0, 10f0]),
+                     material = Material(brdf = DiffuseBRDF(pigment = UniformPigment(WHITE))))]
+        append!(world, cube_vertices, other_spheres, ground, sky)
+        return FlatRenderer(world)
+    elseif renderer_type == "path"
+        ground = [Plane(transformation = translation([0f0, 0f0, -1f0]),
+                        material = Material(brdf = DiffuseBRDF(pigment = CheckeredPigment{4}(color_on = RGB(0.3f0, 0.5f0, 0.1f0), color_off = RGB(0.1f0, 0.2f0, 0.5f0)),
+                                                               reflectance = 0.5f0)))]
+        sky = [Plane(transformation = translation([0f0, 0f0, 100f0]),
+                     material = Material(brdf = DiffuseBRDF(pigment = UniformPigment(BLACK)),
+                                         emitted_radiance = UniformPigment(RGB(1f0, 1f0, 1.2f0))))]
+        # other_spheres = [Sphere(transformation = translation([0.5f0, 0.7f0, 0.1f0]),
+        #                         material = Material(brdf = DiffuseBRDF(pigment = UniformPigment(RGB(0.2f0, 0.7f0, 0.8f0))))),
+        #                  Sphere(transformation = translation([-0.2f0, -0.8f0, -0.8f0]) * scaling(0.5f0),
+        #                         material = Material(brdf = SpecularBRDF(pigment = UniformPigment(RGB(0.6f0, 0.2f0, 0.3f0)))))]
+        other_spheres = [Sphere(transformation = translation([0f0, 0f0, -0.8f0]),
+                                material = Material(brdf = SpecularBRDF(pigment = UniformPigment(RGB(0.55f0, 0.55f0, 0.55f0))))),
+                         Sphere(transformation = translation([4f0, 0f0, -0.2f0]) * scaling(0.8),
+                                material = Material(brdf = SpecularBRDF(pigment = UniformPigment(RGB(0.6f0, 0.2f0, 0.3f0))))),
+                         Sphere(transformation = translation([0f0, 4f0, 0f0]),
+                                material = Material(brdf = DiffuseBRDF(pigment = UniformPigment(RGB(0.2f0, 0.7f0, 0.8f0))))),
+                         Sphere(transformation = translation([0f0, -5f0, 0.5f0]) * scaling(1.5),
+                                material = Material(brdf = SpecularBRDF(pigment = UniformPigment(RGB(07f0, 0.7f0, 0.1f0))))),
+                         Sphere(transformation = translation([-4f0, 0f0, 2f0]) * scaling(0.6),
+                                material = Material(brdf = SpecularBRDF(pigment = UniformPigment(RGB(0.2f0, 0.6f0, 0.3f0)))))]
+        append!(world, ground, sky, other_spheres)
+        return PathTracer(world)
+    else
+        # TODO throw error
+    end
```

Comment:
This is the (now missing) runtime error I was talking about thin [this comment](https://github.com/Paolo97Gll/Raytracer.jl/pull/20/commits/d99df246f17e9a9fdeaaca16e180de52341c9ed0#r646335069)